source/Octopus.Server.Client/Model/MachineBasedResource.cs

```diff
@@ -51,5 +50,8 @@ public abstract class MachineBasedResource : Resource, INamedResource, IHaveSlug
         public string Architecture { get; set; }
         
         public string Slug { get; set; }
+
+        [Writeable]
+        public bool SkipInitialHealthCheck { get; set; }
```

Comment:
The only way I could get this information through the Client is by adding it to the MachineBasedResource. I don't love this but I also feel like the benefits outweigh the drawbacks in this case. Most people won't even care about this property.

I did consider adding a code comment above it to explain that it's only used when creating a new Machine (Deployment Target or Worker).

Reply:
@APErebus I'm interested in your opinion on this.

Reply:
Do machines also have this initial health-check?

Reply:
Yes, I believe they do, Although your comment made me realise I forgot to update the `CreateWorkerCommandHandler` in server.

Reply:
I've updated the Server codebase to also pass this flag down when a worker is created.

It's interesting that the same test doesn't fail very often (it has failed the same way once) when it uses a worker instead of a deployment target. I believe this is because there is a second level of indirection with the event that causes a health check after creating a worker. 

When you create a worker a `MachineCreatedDomainEvent` is created which is handled in `MachineCreatedDomainEventMapper`. This mapper returns a `MachineCreatedEventV2` which is the same event directly raised when creating a deployment target.